app/mailers/prior_authority/feedback_messages/feedback_base.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module PriorAuthority
+  module FeedbackMessages
+    class FeedbackBase
+      def initialize(submission, comment = '')
+        @submission = submission
+        @comment = comment
+      end
+
+      def template
+        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
+      end
+
+      def contents
+        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
+      end
+
+      def recipient
+        @submission.data['provider']['email']
+      end
+
+      protected
+
+      def defendant
+        @submission.data['defendant']
+      end
+
+      def defendant_name
+        "#{defendant['first_name']} #{defendant['last_name']}"
+      end
+
+      def case_reference
+        @submission.data['laa_reference']
+      end
+
+      def ufn
+        @submission.data['ufn']
+      end
+
+      def application_total
+        'TODO'
```

Comment:
I think we've got this, don't we? from ApplicationSummary.formatted_original_total_cost

Reply:
thanks for the pointer, I have added that now, with tests.